data/helloakash1701.json

```
@@ -0,0 +1,20 @@
+{
+  "name": "Akash Mishra",
+  "displayStatsPublic":true,
+  "type":"personal",
+  "bio": "Devops and Open Source **Digital Nomad**. Travel enthusiast Football ",
+  "avatar":"https://github.com/helloakash1701.png",
```

the following are no longer necessary:
```suggestion
  "bio": "Devops and Open Source **Digital Nomad**. Travel enthusiast Football ",
```